libplen/src/player.rs

```diff
@@ -275,6 +280,23 @@ impl Player {
                 self.laser_charge_time = None;
             }
 
+            if self.weapon_is_wielded(PowerUpKind::Missile) && self.cooldown <= 0. {
+                let dir = self.rotation - std::f32::consts::PI / 2.;
+                self.cooldown = constants::PLAYER_COOLDOWN;
```

Comment:
We may want to have a longer CD for missiles than gun. If so, this should probably be changed to something like if `self.last_shot > constants::PLAYER_COOLDOWN`.

We could do that in a separate PR though

Reply:
I think we should do it in a separate one